EIPS/eip-5484.md

```diff
@@ -0,0 +1,101 @@
+---
+eip: 5484
+title: Consensual Soulbound Tokens
+description: Interface for special NFTS with immutable ownership and pre-determined immutable burn authorization
+author: Buzz Cai (@buzzcai)
+discussions-to: https://ethereum-magicians.org/t/eip-5484-consensual-soulbound-tokens/10424
+status: Draft
+type: Standards Track
+category: ERC
+created: 2022-8-17
+requires: 165, 721
+---
+
+
+## Abstract
+This EIP proposes another soulbound token standard. The interface is an extension of [EIP-721](./eip-721.md). Before issuance, both parties (the issuer and the receiver), have to agree on who has the authorization to burn this token. Burn authorization is immutable after declaration. After its issuance, soulbound tokens can't be transferred, but can be burned based on a predetermined immutable burn authorization.  
+
+## Motivation
+The idea of soulbound tokens has gathered significant attention since its publishing. Without a standard interface, however, soulbound token developments are at a discord. It is hard to develop universal services targeting at soulbound tokens without minimal consensus on the implementation of the tokens. 
```

Comment:
```suggestion
The idea of soulbound tokens has gathered significant attention since its publishing. Without a standard interface, however, soulbound tokens are incompatible. It is hard to develop universal services targeting at soulbound tokens without minimal consensus on the implementation of the tokens. 
```